coap/api/transport.go

```diff
@@ -225,21 +225,21 @@ func receive(svc coap.Service, msg *gocoap.Message) *gocoap.Message {
 		return res
 	}
 
-	created, err := ptypes.TimestampProto(time.Now())
+	occured, err := ptypes.TimestampProto(time.Now())
```

Comment:
I would prefer calling this var `timestamp` or just `t`. Mixing "occured" and "created" is confusing. Here you are just taking a simple timestamp recording, so `t` would fit nicely.

Reply:
I renamed back to `created`. We call it like that in all the other services.